code/game/machinery/doors/door.dm

```diff
@@ -272,6 +276,23 @@
 	if(!user.combat_mode && istype(I, /obj/item/fireaxe))
 		try_to_crowbar(I, user, FALSE)
 		return TRUE
+	if(!user.combat_mode && istype(I, /obj/item/stack/sheet/mineral/wood))
+		var/obj/item/stack/sheet/mineral/wood/plank = I
+
+		if(!plank.get_amount() >= 2)
+			balloon_alert(user, "need two [plank] sheets!")
+			return
+
+		balloon_alert(user, "constructing barricade...")
+		playsound(src, 'sound/items/hammering_wood.ogg', 50, TRUE)
+		if(!do_after(user, 5 SECONDS, target = src) || !plank.use(2) || (locate(/obj/structure/barricade/wooden/crude) in loc))
+			return
+
+		balloon_alert(user, "barricade constructed")
+		// mak sure this gets converted to an .ogg (since it's a .wav) and then credit it to soundcloud (public domain)
```

Comment:
?

Reply:
It was a reminder to convert the sound from .wav to .ogg

Removed with https://github.com/tgstation/tgstation/pull/69676/commits/fab5f9bf83832c2a9ff88e16ad9fca9c1e27600c